client/components/UserNotifications/ThreadNotifications.tsx

```diff
@@ -21,7 +21,7 @@ type Props = {
 
 const getAccentColors = (communityAccentColor: string, isRead: boolean) => {
 	const opaque = isRead ? '#aaa' : communityAccentColor;
-	const transparent = Color(opaque).alpha(0.04);
+	const transparent = Color(opaque).alpha(0.04).toString();
```

Comment:
this is to fix a type-error, which became known after adding types for the color package